diff --git a/node_modules/react-native-local-barcode-recognizer/android/build.gradle b/node_modules/react-native-local-barcode-recognizer/android/build.gradle
index c1b99d6..79f90ea 100644
--- a/node_modules/react-native-local-barcode-recognizer/android/build.gradle
+++ b/node_modules/react-native-local-barcode-recognizer/android/build.gradle
@@ -5,19 +5,19 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.4'
+        classpath 'com.android.tools.build:gradle:4.2.2'
     }
 }
 
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 27
-    buildToolsVersion "27.0.3"
+    compileSdkVersion 30
+    buildToolsVersion "30.0.2"
 
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 22
+        targetSdkVersion 30
         versionCode 1
         versionName "1.0"
     }
diff --git a/node_modules/react-native-local-barcode-recognizer/ios/RNReactNativeLocalBarcodeRecognizer.podspec b/node_modules/react-native-local-barcode-recognizer/ios/RNReactNativeLocalBarcodeRecognizer.podspec
index 9462787..22bfa25 100644
--- a/node_modules/react-native-local-barcode-recognizer/ios/RNReactNativeLocalBarcodeRecognizer.podspec
+++ b/node_modules/react-native-local-barcode-recognizer/ios/RNReactNativeLocalBarcodeRecognizer.podspec
@@ -6,7 +6,7 @@ Pod::Spec.new do |s|
   s.description  = <<-DESC
                   RNReactNativeLocalBarcodeRecognizer
                    DESC
-  s.homepage     = ""
+  s.homepage     = "homepage"
   s.license      = "MIT"
   # s.license      = { :type => "MIT", :file => "FILE_LICENSE" }
   s.author             = { "author" => "author@domain.cn" }
@@ -21,4 +21,3 @@ Pod::Spec.new do |s|
 
 end
 
-  
\ No newline at end of file
